### ISSUE TITLE
Dock search bar misaligned in gestural navigation mode

### ISSUE THREAD

**Issue by spezi77** [amirzaidi/Shade#25]
After turning on the gestural navigation mode (Android 10), the search bar looks fine at first. But as soon as you switch to an app and return to the home screen with a swipe up gesture, the search bar is placed too high and touches the dock icons.

If you have the 2- or 3-button navigation mode everything is fine.

I'll add a screenshot.

**Comment by spezi77** [comment]
https://imgur.com/a/HGdn9f4

**Comment by amirzaidi** [comment]
Which phone is this? Are you using some kind of status bar movement mod?

**Comment by spezi77** [comment]
> Which phone is this? Are you using some kind of status bar movement mod?

This is marlin (Pixel XL). I don't use any status bar mods. The issue occurs only while using system gestures (gestural navigation).

**Comment by amirzaidi** [comment]
Does the app drawer still look normal when this bug happens? Can you try force closing the launcher after enabling gestures and seeing if the bug still happens?

**Comment by spezi77** [comment]
> Does the app drawer still look normal when this bug happens? Can you try force closing the launcher after enabling gestures and seeing if the bug still happens?

Yes, app drawer is fine. The layout of the app search bar is as expected.

**Comment by spezi77** [comment]
After doing a force close the layout is normal again. But as soon as you open an app and swipe up, the layout glitch (described above) can be observed again.
